### PR TITLE
[improvement](alter table) cann't rewrite force property

### PR DESCRIPTION
**Pull request by yujun777**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by yujun777** [comment]
run buildall

**Review by deardeng — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/catalog/InternalSchemaInitializer.java [hunk 1]
```diff
@@ -92,8 +92,9 @@ public void run() {
 
     @VisibleForTesting
     public static void modifyTblReplicaCount(Database database, String tblName) {
-        if (!(Config.min_replication_num_per_tablet < StatisticConstants.STATISTIC_INTERNAL_TABLE_REPLICA_NUM
-                && Config.max_replication_num_per_tablet >= StatisticConstants.STATISTIC_INTERNAL_TABLE_REPLICA_NUM)) {
+        if (Config.isCloudMode()
+                || Config.min_replication_num_per_tablet >= StatisticConstants.STATISTIC_INTERNAL_TABLE_REPLICA_NUM
+                || Config.max_replication_num_per_tablet < StatisticConstants.STATISTIC_INTERNAL_TABLE_REPLICA_NUM) {
```
Comment (924060929) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/InternalSchemaInitializer.java:97]: This modification looks like wrong

**Pull request merged**